lib/node.js

```diff
@@ -896,8 +896,34 @@ declare module "querystring" {
   declare function unescape(str: string, decodeSpaces?: boolean): string;
 }
 
+declare class readline$Interface extends events$EventEmitter {
+  close(): void;
+  pause(): void;
+  prompt(preserveCursor?: boolean): void;
+  question(query: string, callback: (answer: string) => void): void;
+  resume(): void;
+  setPrompt(prompt: string): void;
+  write(val: string | void | null, key?: {
+    name: string,
+    ctrl?: boolean,
+    shift?: boolean,
+    meta?: boolean
+  }): void;
+}
+
 declare module "readline" {
-  // TODO
+  declare var Interface : typeof readline$Interface;
+  declare function clearLine(stream: stream$Writable, dir: number /* -1 | 1 | 0 */): void;
```

Comment:
Flow wouldn't let me put a negative number literal here which I assume is a bug since positive ones are allowed?